installer/lib/phx_new/web.ex

```diff
@@ -58,12 +58,9 @@ defmodule Phx.New.Web do
     {:eex,  "phx_test/views/page_view_test.exs",              :web, "test/:web_app/views/page_view_test.exs"},
   ]
 
-  template :bare, [
-    {:text, "phx_assets/bare/gitignore", :web, ".gitignore"},
-  ]
+  template :bare, []
```

Comment:
I was not sure if leaving or removing this line.

Other option is to mimic the directory structure for single apps and create `brunch` and `bare` directories with two `gitignore` in them.